custom/abt/reports/supervisory_datasource_indicators.json

```diff
@@ -0,0 +1,225 @@
+[
```

Comment:
Oh awesome, I didn't know that existed. I was copy and pasting the indicator files. I put them in the repo incase I wanted to review the changes. Is there documentation on how to use `CustomDataSourceConfiguration`s?